pandas/tests/series/test_indexing.py

```diff
@@ -1324,6 +1324,13 @@ def test_timedelta_assignment(self):
         s.loc['A'] = timedelta(1)
         tm.assert_series_equal(s, expected)
 
+        # GH 14155
+        s = Series(10*[np.timedelta64(10, 'm')])
+        s.ix[[1,2,3]] = np.timedelta64(20, 'm')
+        expected = pd.Series(10*[np.timedelta64(10, 'm')])
+        expected.ix[[1,2,3]] = pd.Timedelta(np.timedelta64(20, 'm'))
+        tm.assert_series_equal(s, expected)
```

Comment:
You will also have to clean those lines above for PEP8 issues (eg whitespace around operators or after comma's). Travis will also complain (by a failing lint test) about this.